client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -400,7 +400,8 @@ function getNotes(
           !versionIsPreview(item.version_added, browser)
             ? {
                 iconName: "footnote",
-                label: "Full support",
+                label:
+                  "Full support (earliest supporting release version unknown)",
```

Comment:
IIRC this is what shows when a feature has a version_added and no other information. You want to show this message only in the case when `item.version_added === true` .

Reply:
Can you please provide a link where I can see Yes. The link mentioned in bug does not show it.

Reply:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/user_scripts#browser_compatibility or any found from this search: https://github.com/search?q=repo%3Amdn%2Fbrowser-compat-data+"%5C"version_added%5C"%3A+true"&type=code

Reply:
The number of tables affected is likely to decrease further and perhaps even eventually become a non-issue as people make a concerted effort to be more precise with the data in https://github.com/mdn/browser-compat-data.

Reply:
thanks. Updated PR.